src/table/table.scss

```diff
@@ -122,265 +251,316 @@
         }
       }
 
-      &.iui-tables-head {
-        @include themed {
-          background-color: t(iui-color-background-3);
-        }
-        user-select: none;
+      // #region Indented rows
+      &.iui-expander-indent-0 {
+        padding-left: 0;
+      }
 
-        .iui-icon {
-          @include themed {
-            fill: t(iui-icons-color-actionable);
-          }
+      &.iui-expander-indent-1 {
+        padding-left: $iui-xl;
+      }
 
-          &:hover {
-            @include themed {
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+      &.iui-expander-indent-2 {
+        padding-left: $iui-xl * 2;
+      }
 
-        > .iui-filter {
-          margin-left: $iui-sm;
-          cursor: pointer;
+      &.iui-indent-1 {
+        padding-left: $iui-xl + $iui-s;
+      }
 
-          > .iui-icon {
-            width: $iui-icons-default;
-            height: $iui-icons-default;
-          }
-        }
+      &.iui-indent-2 {
+        padding-left: ($iui-xl + $iui-s) * 2;
+      }
+      // #endregion Indented rows
+    }
 
-        > .iui-sort {
-          flex-grow: 1;
-          justify-content: flex-end !important;
+    // #region Selection
+    &:not(.iui-selected) + .iui-selected {
+      padding-bottom: $iui-xxs;
+      border-bottom: none;
+    }
 
-          > .iui-icon-wrapper {
-            cursor: pointer;
+    &.iui-selected {
+      padding: 0;
+      border-bottom: none;
+      @include themed {
+        border: solid 1px t(iui-color-foreground-primary);
+        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+      }
 
-            > .iui-icon {
-              width: $iui-icons-default;
-              height: $iui-icons-default;
-            }
-          }
+      &:hover {
```

Comment:
selected row gets hover?
I dont really see it..

Reply:
I have no hover effect on selected rows in this PR.  I know our current table does have a hover effect on selected rows.

Reply:
I think I had a duplicate rules in the code and this is the rule that should be in place : 
```scss
&:hover {
	background: rgba(
		t(iui-color-foreground-primary-rgb),
		t(iui-opacity-6)
	);
}

```

Reply:
Sorry 2 rules : 
```scss
&.iui-selected {
	&:hover { // this is for selected line
		background: rgba(
			t(iui-color-foreground-primary-rgb),
			t(iui-opacity-5)
		);
	}
}
...
// #endregion Selection
&:hover { // This is for white line
	background-color: rgba(
		t(iui-color-foreground-primary-rgb),
		t(iui-opacity-6)
	) !important;
}
```